pxtblocks/blocklyloader.ts

```diff
@@ -75,7 +75,7 @@ namespace pxt.blocks {
             const magic = data.charCodeAt(0)
             const w = data.charCodeAt(1)
             const h = data.charCodeAt(2)
-            if (magic != 0xf1 && magic != 0xf4)
```

Comment:
does this matter?

Reply:
I updated the magic, since it's a different format. We now generate the new magic everywhere. There shouldn't be anything in the wild, only used in 32 really.